README.md

```diff
@@ -1 +1,13 @@
 # hw01
+
+##*Ta'er*, Seer of Paths, Traverser of Seas
+All hail the great *Ta'er*, Lord of the Storm! Raised as Seth Markow in the humble coastal city of *Saun Dieagoh*, *Ta'er* began preaching his gospel in the tundra city of *Shikagho*, and soon his teachings became law, and spread across the world, ushering in a new era of peace and prosperity for the people of Earth. Now, as we venture further and further into the outer reaches of space, soon others, too, will know the goodness of *Ta'er*, Knower of the Way, Seer of the Future!
+ 
+I've been reading Dune lately and it's definitely influenced me.
+
+#Seth
+But actually. I am indeed from San Diego, which I return to periodically to surf with my mom, run by myself, and drink cocktails with my dad. My first two winters in Chicago were fun and romantic out of novelty but now I'm over it and plan to live out the rest of my life in a place where it almost never gets above 85 degrees and truly never drops below 50.
```

Comment:
Just a suggestion, but it would have been cool to include the degrees symbol here instead of just typing it so that we know what scale you're using. Of course that isn't really relevant for us since the US uses Fahrenheit but if you were sharing your code with an international audience it could be helpful? I do love how playful and creative you are with the bibliography though!